src/core/layout/navbar/navigation/cameras/styles.js

```diff
@@ -0,0 +1,3 @@
+import { makeStyles } from '@material-ui/core'
+
+export const useStyles = makeStyles({ root: { color: 'white', paddingLeft: '5px' } })
```

Comment:
This is the `style.js` used by the `detection` link:
![image](https://user-images.githubusercontent.com/23569938/128638634-4bb9b676-e45f-482e-947b-46637da0c964.png)

And this is the one used the `home` link:
![image](https://user-images.githubusercontent.com/23569938/128638659-e50333e5-d3d7-4c8a-aa30-697da3a227fc.png)

As you can see, these three are incredibly similar. Maybe we can use a single `style` across our links. If so, we can also remove the directories holding each link and replace them with regular `.jsx` files.

Reply:
Done